modules/raycast/SCsub

```diff
@@ -67,8 +67,9 @@ if env["builtin_embree"]:
     env_raycast.AppendUnique(CPPDEFINES=["NDEBUG"])  # No assert() even in debug builds.
 
     if not env.msvc:
+        # TODO: Re-evaluate the need for this / streamline with common config.
         if env["arch"] in ["x86_64", "x86_32"]:
-            env_raycast.Append(CCFLAGS=["-msse2", "-mxsave"])
+            env_raycast.Append(CCFLAGS=["-mxsave"])
```

Comment:
Are we fine no longer passing `-msse2` to `x86_64` builds here?

Reply:
`-msse2` is implied for x86_64, there are no x86_64 cpus without SSE2!